.github/ISSUE_TEMPLATE/2-bug-report.md

```diff
@@ -7,20 +7,47 @@ assignees: ''
 
 ---
 
-<!---
-Please do not ask for help or general inquiry here, use RPCS3 Discord server instead.
```

Comment:
This should probably be preserved somehow, to avoid support requests on tracker

Reply:
It was commented out for some reason. So I decided to remove it completely with redesign.
Maybe it's worth preserving as 0 point in the list?

Reply:
It was commented out in order to show on the beginning of the issue template as soon as you create a new issue but not be visible if you posted the issue without removing the comment

Reply:
Unsure what the best way to add it is, probably adding as point 0 suffices

Reply:
Ok, will try it now.